inc/themes/core.base/frontend/templates/index/index.twig

```diff
@@ -66,8 +68,14 @@
                 <p class="board-statistics__body">
                     {{ lang.stats_posts_threads }}<br />
                     {{ lang.stats_numusers }}<br />
-                    {{ lang.stats_newestuser|raw }}<br />
-                    {{ lang.stats_mostonline|raw }}
+                    {{ lang.stats_newestuser }},
+                    {% if newestmember == 0 %}
+                        {{ lang.nobody }}
+                    {% else %}
+                        <a href="{{ get_profile_link(newestmember) }}">{{ newestmember|format_name }}</a>
```

Comment:
Why not add a helper for `build_profile_link()`? I'm concerned about the inconsistencies on formatted built links.

Reply:
Yes, it would be implemented separately.

Alternatively, maybe the decrease of usage of the `build_profile_link` may be the desired approach instead of the contrary.

Since we are trying to move some basic logic to templates, the way you are doing it seems reasonable.